spec/models/chat_message_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ChatMessage do
+  describe '.cook' do
+    it 'does not support HTML tags' do
```

Comment:
Test for markdown rules which we do not want is not meant to be exhaustive but I thought it'll be good to test for some of the basic rules that does not make sense for chat.